docs/concepts/gateway/customization.md

```diff
@@ -259,6 +259,44 @@ Jina provides the Gateway with a list of ports and protocols to expose. Therefor
 on multiple ports using different protocols.
 ```
 
+(executor-streamer)=
+## Calling an Individual Executor with {class}`~jina.serve.streamer.ExecutorStreamer`
+{class}`~jina.serve.streamer.ExecutorStreamer` allows you to interface with individual Executors from the Gateway. An instance of 
+this class knows where each Executor lives and allows you to call them individually. 
+An `executor` object (instance of {class}`~jina.serve.streamer.ExecutorStreamer`) is injected by Jina to your gateway class.
+
+After transforming requests that arrive to the gateway server into Documents, you can call the Executor in your Python code 
+using `self.executor['executor_name'](args)`.
```

Comment:
the property should be `plural`

Reply:
Oh but it is only one `ExecutorStreamer` though. The one class knows about all the executors

Reply:
Removed references to the class as suggested so this is not longer relevant I think